--- a/build
+++ b/build
@@ -26,9 +26,6 @@ PNG_DIR="$SCRIPT_DIR/pngs"
 SVGS_DIR="$SCRIPT_DIR/svgs"
 HL_DIR="$SCRIPT_DIR/hl"
 
-AUTHORS="$SCRIPT_DIR/AUTHORS"
-LICENSE="$SCRIPT_DIR/LICENSE"
-
 ACCENTS="blue dark flamingo green lavender light maroon mauve peach pink red rosewater sapphire sky teal yellow"
 
 while getopts "f:a:" option; do
@@ -62,8 +59,6 @@ for accent in $ACCENTS; do
 
   scripts/build-cursors "$theme_src_dir" "$theme_png_dir" "$theme_hl_dir" "$theme_out_dir"
 
-  cp -f "$AUTHORS" "$theme_out_dir"
-  cp -f "$LICENSE" "$theme_out_dir"
   cp -f "$theme_src_dir/index.theme" "$theme_out_dir"
   cp -f "$theme_hl_dir/manifest.hl" "$theme_out_dir"
 done
