diff --git a/src/templates/index.theme.tera b/src/templates/index.theme.tera
index 52619fda..13acafec 100644
--- a/src/templates/index.theme.tera
+++ b/src/templates/index.theme.tera
@@ -1,6 +1,6 @@
 ---
 whiskers:
-  version: 2.5.1
+  version: ^2.5.1
   matrix:
   - accent: ["blue", "dark", "flamingo", "green", "lavender", "light", "maroon", "mauve", "peach", "pink", "red", "rosewater", "sapphire", "sky", "teal", "yellow"]
   - flavor
diff --git a/src/templates/manifest.hl.tera b/src/templates/manifest.hl.tera
index bd54e1c9..904e1916 100644
--- a/src/templates/manifest.hl.tera
+++ b/src/templates/manifest.hl.tera
@@ -1,6 +1,6 @@
 ---
 whiskers:
-  version: 2.5.1
+  version: ^2.5.1
   matrix:
   - accent: ["blue", "dark", "flamingo", "green", "lavender", "light", "maroon", "mauve", "peach", "pink", "red", "rosewater", "sapphire", "sky", "teal", "yellow"]
   - flavor
diff --git a/src/templates/svgs.tera b/src/templates/svgs.tera
index 53bd608b..40a9eaac 100644
--- a/src/templates/svgs.tera
+++ b/src/templates/svgs.tera
@@ -1,6 +1,6 @@
 ---
 whiskers:
-  version: 2.5.1
+  version: ^2.5.1
   matrix:
     - file: ['alias', 'all-scroll', 'bottom_left_corner', 'bottom_right_corner', 'bottom_side', 'cell', 'center_ptr', 'color-picker','col-resize', 'context-menu',
     'copy', 'crosshair', 'default', 'dnd-move', 'dnd-no-drop', 'down-arrow', 'draft', 'help','left-arrow', 'left_side', 'no-drop', 'not-allowed', 'openhand',
